docs/versioning.rst

```diff
@@ -0,0 +1,96 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+When a new major version is released it becomes the current release line, where all new features, bug fixes and security patches land.
+
+
+.. list-table:: Release Lines
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Line
+     - Supported Runtimes
+     - Minimum Trace Agent
+     - Status
+   * - 0.x
+     - CPython: 2.7, 3.5+
+     - 
+     - MAINTENANCE [2022-02-25]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.22
+     - CURRENT
+   * - 2.x
+     - TBD
+     - TBD
+     - DEVELOPMENT
+
+
+The **status** of each release line determines when changes are made:
+
+MAINTENANCE
+    No longer under development.
+
+    Receives all bug fixes and security fixes.
+
+    Will not receive any new features.
+
+CURRENT
+    Under active development.
+
+    Receives all backwards compatible new features, bug fixes, and security fixes.
+
+DEVELOPMENT
+    Under active development.
+
+    Receives all incompatible new features.
+
+    Not yet released or maintained.
```

Comment:
```suggestion
    Under active development.

    Receives all incompatible new features.

    Not yet released or maintained.
    
    Will become the next current version when released.
```

Do we want to note somewhere about how as soon as a new major release line is released, the CURRENT will become MAINTENANCE?